todo_list_app/tests/test_auth_apis.py

```diff
@@ -0,0 +1,41 @@
+
+import json
+from rest_framework.test import APIClient
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from rest_framework import status
+
+User = get_user_model()
+
+
+class TestAuthAPI(TestCase):
```

Comment:
1. `setUp` should only set up objects or functions required by any test. Any API calls or processing that is part of the test should be in the test itself or if it's common among more than one test you can create a helper private function something like `def _register_user(data)`.
2. How to mock or patch a function and an object to get the return value of it.
3. `auth` is just authentication of user and registration is a separate process.
4. Update folder structure and keep directories and files inside the `tests` directory the same as it is outside. We can rename them to show that they are tests either by prefixing or postfixing the test keyword.